server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java

```diff
@@ -76,6 +76,17 @@ public void tearDown() throws Exception {
     }
 
     @Test
+    public void testStoreAndGetMappings() throws
+        org.apache.james.rrt.api.RecipientRewriteTable.ErrorMappingException, RecipientRewriteTableException {
```

Comment:
why FQDN?

Reply:
Good question I just copy and paste an already existing test, I will check why